concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountTransaction.java

```diff
@@ -1,35 +1,108 @@
 package com.concordium.sdk.transactions;
 
-import io.grpc.netty.shaded.io.netty.util.internal.UnstableApi;
 import lombok.*;
 
 import java.nio.ByteBuffer;
 
-@EqualsAndHashCode(callSuper = true)
+import static com.google.common.primitives.Bytes.concat;
+
 @Getter
+@EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
-public final class AccountTransaction extends AbstractAccountTransaction {
+public class AccountTransaction extends BlockItem {
```

Comment:
Can't we now extend the `public static AccountTransaction fromBytes(ByteBuffer source)` function on this class?

Reply:
`fromBytes` is a static function. It has not been removed because its a public function and removing might break code. 
but its not being used anywhere and should not be used.